### PR TITLE
감성분석 차트 변경

### PR DESCRIPTION
**Pull request by minji0916**
기준선 회색으로 변경
감정적으로 두드러진 메시지 표시
오늘의 환자 기분 표시

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -31,14 +31,28 @@ def get_voicetext():
...
+            'Uh , sure , so , um , I woke up probably three or four days ago , which , uh , wheezing and short of breath .',
+            'I cough infrequently , but no , uh , chest pain .',
+            'Uh , no , and I also took a test , which was negative .',
+            'Uh , it has been getting a lot worse .',
+            'Um , I had a gone hiking , and I got caught in the rain the day before this all started .'
+        ],
+        'senti_scores': [0.337, -0.671, 0.0, 0.069, -0.952, -0.231],
+        'senti_magnitudes': [0.379, 0.724, 0.3, 0.286, 0.987, 0.257]
+    }]
+    """
```
Comment (dealim) [inline app.py:55]: 이거 잘못짬

**Comment by minji0916** [comment]
뭐라했는데?

File: templates/show_text_emotion_analysis.html [hunk 2]
```diff
@@ -97,43 +97,43 @@ <h1>오늘의 환자 기분</h1>
...
+    // 긍정, 부정 기준선을 그래프에 그리기 위해 리스트에 문장 개수만큼 채워 넣음
+    var positiveBaseline = new Array(sentiScores.length).fill(0.25);
+    var negativeBaseline = new Array(sentiScores.length).fill(-0.25);
 
     // labels를 sentence 개수만큼 만듦 => text1, text2, ...
     var labels = []
     for (var i = 1; i <= sentences.length; i++) {
-        labels.push("text" + i);
+          labels.push("");
     }
```
Comment (dealim) [inline templates/show_text_emotion_analysis.html:108]: 다른거

File: templates/show_text_emotion_analysis.html [hunk 3]
```diff
@@ -32,43 +97,43 @@
     var overallScore = {{ patient.doc_sentiment_score | tojson }};
     var overallMagnitude = {{ patient.doc_sentiment_magnitude | tojson }};
 
-    // 전체적인 값은 기준선으로 그래프에 그리기 위해 리스트에 문장 개수만큼 채워 넣음
-    var overallScoreList = new Array(sentiScores.length).fill(overallScore);
-    var overallMagnitudeList = new Array(sentiScores.length).fill(overallMagnitude);
+    // 긍정, 부정 기준선을 그래프에 그리기 위해 리스트에 문장 개수만큼 채워 넣음
+    var positiveBaseline = new Array(sentiScores.length).fill(0.25);
+    var negativeBaseline = new Array(sentiScores.length).fill(-0.25);
```
Comment (dealim) [inline templates/show_text_emotion_analysis.html:103]: 리뷰를 시작

**Review by dealim — CHANGES_REQUESTED** [review]
이거 잘못했으니 바꿔줘요
